microsoftgraph/microsoft-graph-explorer-v2
# Fix: display /$metadata response

## Overview

Running the /$metadata request was not displaying the response causing the app to hang
Now shows the response in form of xml
Refactors the display of components based on the header 'content-type' and removes magic strings

Fixes #292 

### Demo

![image](https://user-images.githubusercontent.com/53429535/70236450-fce99380-1775-11ea-872e-ac7dafcd9c3c.png)


### Notes

Monaco editor currently does not have a formatter for the xml so I added a function helper.

## Testing Instructions

* Run a /$metadata request
* Notice the meta data displayed in a pretty formatted response
